### ISSUE TITLE
Medical rewrite: How to go basic?

### ISSUE THREAD

**Issue by IllidanS4** [acemod/ACE3#7367]
We have been using basic medical system before the new update, but now it seems the change unified the two systems and made it more configurable. How to configure the addon to make it work like before? Is it possible to ignore blood pressure, pulse etc. to revive with adrenaline like before? What are the previous basic rates of blood loss, transfusion etc.? How to make death almost impossible, like with basic before?

**Comment by yurivw** [comment]
The only way I've found to stop people from bleeding out is to stop bleeding entirely. This isn't a very good solution but it's the best one I could find.

**Comment by Arcanum417** [comment]
So basicly this https://github.com/acemod/ACE3/pull/7060#issuecomment-509855005

**Comment by IllidanS4** [comment]
I am sorry but so far the update is very disappointing. There seems to be no clear guide to how to set the options to work like before, if that is even possible: pain cannot be fully removed without PAK (and morphine cannot be set to work instantly), cardiac arrest happens way too often even if you aren't wounded that much, blood loss is very quick and death can happen even if you try your best to heal somebody. And considering the bugs, like surviving when you explode in a vehicle without any damage, or that CPR sometimes "half-revives" a person (black screen but you can look around and use ACE menu) makes the useful features (the medical menu looks especially nice) of the update really not worth it, and we are considering downgrading to the previous version for the meantime.

**Comment by JinougaF** [comment]
Some examples:

ace_medical_statemachine_fatalInjuriesPlayer = 2; (Players can never be killed off completely, they remain in cardiac arrest (revive state))

ace_medical_statemachine_cardiacArrestTime = 600; (Cardiac arrest time in seconds, basically becomes your former revive timer)

ace_medical_spontaneousWakeUpEpinephrineBoost = 30; (Pretty much makes sure that using epi will bring someone with stable vitals up)

I havent test all these settings in a real mission, just have a test with another player in a test scenario, but it seem met our goal. So for ouur goal, our unit want a revive system but could prevent player died so often, what we did in older medical system is like player can not be instantly killed by any means, cuz we are a small unit with only like 4-5 peaple squad for spec ops missions, so we would not have heavy fire fight, we disable instantly death mainly to prevent some bugs that would kill players by accidents, i.e. Zeus accidently blow up a vehicle near players.

So for from our test scenario, when we set the medical system as above, no player would died by any ways of direct hit, even direct hit by a tank shell player would downed wait for revive but not died instantly. However the player still would bleed out, to prevent this we also changed the bleed speed and amount( sry i kind forgot which part in the addon setting it that), and it seem work as the older medical system.
